truffle/src/main/ruby/core/rbconfig.rb

```diff
@@ -72,18 +72,14 @@ module RbConfig
       'LIBRUBY_A' => '',
       'LIBRUBYARG' => '',
       'prefix' => '',
-      'ruby_install_name' => 'jruby-truffle',
+      'ruby_install_name' => ruby_install_name,
       'RUBY_SO_NAME' => '$(RUBY_BASE_NAME)'
   }
   
   ruby_home = Truffle::Boot.ruby_home
 
   if ruby_home
-    bindir = if ruby_home.end_with?('/mxbuild/ruby-zip-extracted')
-               File.expand_path('../../bin', ruby_home)
-             else
-               "#{ruby_home}/bin"
-             end
+    bindir = "#{ruby_home}/bin"
```

Comment:
Because mx extracts the lib to somewhere else? Right... The GraalVM launcher can't delegate like that - it's actually a Java program that does some other work before calling our `main`.